install/fedora.md

```diff
@@ -1,5 +1,53 @@
 Installing CJDNS on Fedora
 ==========================
+
+Cjdns was added to the Fedora repository beginning with Fedora 23.  It is in the testing repository until it gains sufficient positive karma.  To install:
+
+```bash
+sudo dnf install cjdns cjdns-tools cjdns-selinux --enablerepo=updates-testing
+```
+
+The cjdns-tools package has peerStats and other nodejs tools.  Python versions of the tools are in cjdns-python - although the ones conflicting with the nodejs tools are not symlinked to /bin.
+
+The cjdns-selinux package has an selinux sandbox for cjdroute, to prevent it from doing anything wonky through accident or malice.  Some don't trust selinux (because of NSA origins), but it is installed by default on Fedora - so if you are running Fedora you want this.
+
+Start cjdroute:
+
+```bash
+sudo systemctl enable cjdns #This sets cjdns to be started on boot. if you don't want that, feel free to leave this line out.
+sudo systemctl start cjdns #This actually starts cjdroute.
+````
+
+Checking the logs:
+```bash
+sudo systemctl status -l cjdns
+```
+
+Starting cjdroute for the first time will generate a new config file in ```/etc/cjdroute.conf```.  The default config will Just Work if there are any other cjdns nodes within reach of ethernet broadcast - for example on your local LAN or mesh.  Otherwise, you will need to add some internet peers as discussed elsewhere.  
+
+After changing anything in the configuration, restart cjdns:
+
+```bash
+sudo systemctl restart cjdns
+```
+
+If you are on a laptop, you may find that after waking up from suspend or hibernate, cjdroute takes a minute or two to make coffee and figure out what just happened.  To speed this up, just restart cjdroute.  To always restart cjdroute upon awakening, enable the cjdns-resume service:
+
+```bash
+sudo systemctl enable cjdns-resume
```

Comment:
Could we also get this service into `cjdns/contrib/systemd` or is it too distro specific?